core/shared/src/main/scala/fs2/interop/flow/package.scala

```diff
@@ -199,6 +198,51 @@ package object flow {
   ): Stream[F, Nothing] =
     StreamSubscription.subscribe(stream, subscriber)
 
+  /** Creates a [[Processor]] from a [[Pipe]].
+    *
+    * You are required to manually subscribe this [[Processor]] to an upstream [[Publisher]], and have at least one downstream [[Subscriber]] subscribe to the [[Consumer]].
+    *
+    * Closing the [[Resource]] means not accepting new subscriptions,
+    * but waiting for all active ones to finish consuming.
+    * Canceling the [[Resource.use]] means gracefully shutting down all active subscriptions.
+    * Thus, no more elements will be published.
+    *
+    * @see [[unsafePipeToProcessor]] for an unsafe version that returns a plain [[Processor]].
+    *
+    * @param pipe The [[Pipe]] which represents the [[Processor]] logic.
+    * @param chunkSize setup the number of elements asked each time from the upstream [[Publisher]].
+    *                  A high number may be useful if the publisher is triggering from IO,
+    *                  like requesting elements from a database.
+    *                  A high number will also lead to more elements in memory.
+    */
+  def pipeToProcessor[F[_], I, O](
```

Comment:
Actually I forgot, we (me 😅 ) deprecated this entire package. So actually we don't need to add these here at all, or if we decide to they can be private so as not to increase the API surface? Idk, not a huge deal.

Reply:
Wait, why? This `package object` is not / should not be deprecated!
This is the API we are exposing.

Reply:
> but what about the other side of the syntax? Also, what about methods that are not syntax?

It should be possible to implement everything as either implicit syntax (not requiring imports) or as methods on the relevant companion objects. I think syntax requiring imports is more of an anti-pattern.